admin/index.html

```diff
@@ -350,10 +351,15 @@
                     <td><input id="extraSet" type="checkbox" class="value"/></td>
                     <td></td>
                 </tr>
+                <tr>
+                    <td><label class="translate" for="sendStateObject">Send state object as mqtt message 2222</label></td>
+                    <td><input id="sendStateObject" type="checkbox" class="value"/></td>
+                    <td></td>
+                </tr>
                 <tr>
                     <td><label class="translate" for="maxTopicLength">Max topic length:</label></td>
-                    <td><input id="maxTopicLength" class="value" size="17"/></td>
-                    <td class="translate">chars</td>
+                    <td><input id="maxTopicLength" type="checkbox" class="value"/></td>

```

Comment:
I don't understand this. Why maxTopicLength is now checkbox.
Admin2 is no more active, but anyway.

Reply:
It was a paste & copy mistake by the new line above. I undo that one of course.